pkg/apis/ndm/v1alpha1/types.go

```diff
@@ -13,17 +13,17 @@ import (
 // StoragePoolClaim describes a StoragePoolClaim.
 type StoragePoolClaim struct {
 	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata",omitempty`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Spec StoragePoolClaimSpec `json:"spec"`
 }
 
 // StoragePoolClaimSpec is the spec for a StoragePoolClaimSpec resource
 type StoragePoolClaimSpec struct {
-	Name       string `json:"name"` // disk name (e.g. sdb, sdc)
-	Format     string `json:"format"` // filesystem format (e.g. ext3, ext4)
+	Name       string `json:"name"`       // disk name (e.g. sdb, sdc)
```

Comment:
How can this name work? if you add a USB disk the ordering will change after next boot. I.e your SDB might have become SDC -- we need to use unique names here